### PR TITLE
companion for jsonrpsee v0.10

### PR DESCRIPTION
**Pull request by dvdplm**
Preparation PR for when `jsonrpsee` v0.10 is released.

### CODE REVIEW

File: client/cli/src/commands/run_cmd.rs [hunk 1]
```diff
@@ -100,18 +100,28 @@ pub struct RunCmd {
 	#[clap(long)]
 	pub unsafe_ws_external: bool,
 
-	/// Set the the maximum RPC payload size for both requests and responses (both http and ws), in
-	/// megabytes. Default is 15MiB.
+	/// DEPERECATED, this has no affect anymore. Use `rpc_max_request_size` or
```
Comment (dvdplm) [inline client/cli/src/commands/run_cmd.rs:103]:
```suggestion
	/// DEPRECATED, this has no affect anymore. Use `rpc_max_request_size` or
```

Is there automation in place to show this message to end users?

File: client/cli/src/commands/run_cmd.rs [hunk 2]
```diff
@@ -100,18 +100,28 @@ pub struct RunCmd {
...
+	pub rpc_max_response_size: Option<usize>,
+
 	/// Expose Prometheus exporter on all interfaces.
 	///
 	/// Default is local.
 	#[clap(long)]
 	pub prometheus_external: bool,
 
-	/// Specify IPC RPC server path
+	/// DEPERECATED, IPC support has been removed.
```
Comment (dvdplm) [inline client/cli/src/commands/run_cmd.rs:124]:
```suggestion
	/// DEPRECATED, IPC support has been removed.
```

File: client/cli/src/commands/run_cmd.rs [hunk 3]
```diff
@@ -127,7 +137,7 @@ pub struct RunCmd {
 	#[clap(long, value_name = "COUNT")]
 	pub ws_max_connections: Option<usize>,
 
-	/// Set the the maximum WebSocket output buffer size in MiB. Default is 16.
+	/// DEPERECATED, this has no affect anymore. Use `rpc_max_response_size` instead.
```
Comment (dvdplm) [inline client/cli/src/commands/run_cmd.rs:140]:
```suggestion
	/// DEPRECATED, this has no affect anymore. Use `rpc_max_response_size` instead.
```

File: client/cli/src/commands/run_cmd.rs (hunk 1 above)
Comment (niklasad1) [reply client/cli/src/commands/run_cmd.rs]: no unfortunately not what I'm aware of.

**Pull request merged**